sql/core/src/test/resources/tpcds-plan-stability/approved-plans-v1_4/q11.sf100/explain.txt

```diff
@@ -89,394 +89,394 @@ TakeOrderedAndProject (87)
 
 
 (1) Scan parquet default.store_sales
-Output [4]: [ss_sold_date_sk#1, ss_customer_sk#2, ss_ext_discount_amt#3, ss_ext_list_price#4]
+Output [4]: [ss_sold_date_sk#x, ss_customer_sk#x, ss_ext_discount_amt#x, ss_ext_list_price#x]
 Batched: true
-Location: InMemoryFileIndex [file:/Users/yi.wu/IdeaProjects/spark/sql/core/spark-warehouse/org.apache.spark.sql.TPCDSV1_4_PlanStabilityWithStatsSuite/store_sales]
```

Comment:
How did the tests pass before?

Reply:
the test only checks the simplified form, the explain result is only for debugging purpose.

Reply:
Hm, how do we check the differences by other changes but not caught by this test?
I think we should only result simplified forms here if we test simplified forms.

Reply:
we only want to track the plan changes that may affect performance, and the simplified form is good enough for this purpose. The explain result is only for convenience, so that you don't need to run EXPLAIN yourself when debugging.